lib/servers/datavault/data_vault.py

```diff
@@ -1148,6 +1159,19 @@ def get(self, c, limit=None, startOver=False):
         dataset.keepStreaming(c.ID, c['filepos'])
         return data
 
+    ### Add in saving camera images as a .npy with the dataset
+
+    @setting(22, data='*i', image_size='*i', repetitions='i', returns = '')
+    def save_image(self, c, data, image_size, repetitions):
```

Comment:
so my plan is to steal utility from the haeffner group as well as keep the grapher compatible to both groups so this function name cant be different. Ill add info to the docstring